app/src/pages/inside/common/expandedOptions/expandedOptions.tsx

```diff
@@ -61,6 +62,8 @@ export const ExpandedOptions = ({
 }: ExpandedOptionsProps) => {
   const { formatMessage } = useIntl();
 
+  const { expandedIds, onToggleFolder } = useStorageFolders(instanceKey);
```

Comment:
Let's keep them grouped (feel free to resolve yourself)
```suggestion
  const { formatMessage } = useIntl();
  const { expandedIds, onToggleFolder } = useStorageFolders(instanceKey);
```